### DISCUSSION TITLE
Frigate+ true & false positive

### DISCUSSION THREAD

**Discussion by mabed-fr** [blakeblackshear/frigate#9357] [General]
Hello,

I use frigate from 1 month with 1 cam with same angle.
In this camera there are often 3 cars and for each true/false positive should I label each car and each license plate while the true/false positive is a dog or a cat?

**Comment by NickM-27** [comment]
yes, for every image that you submit you should label every single object before verifying

**Reply by mabed-fr** [reply]
Do you think that prelabeling is possible with a tool on the user side to avoid using too many resources on the frigate side? Do you have any solutions (or ideas) to speed up each submission or verification?

**Reply by NickM-27** [reply]
From what I have heard there are a number of improvements coming to the editor that will speed things up. and yes, there is already a pinned feature request that asks to use the model to do a first pass labeling

**Comment by mabed-fr** [comment]
This is great news! I will look for the reference
